subprojects/docs/src/docs/release/notes.md

```diff
@@ -100,6 +100,22 @@ This means that you can query for the Javadoc or sources _variant_ of a module a
 This also works in multi-projects.
 If activated, a Java and Java Library project automatically provides the `javadocJar` and `sourcesJar` tasks.
 
+## Problems with task definitions called out during build
```

Comment:
We should have something in the user manual as well, calling out that the validation is done at runtime. The user manual would also be a good place to show a sample warning. When we start failing on those warnings, we can adjust the user manual to show the failure.

Reply:
The deprecation itself needs to go into the upgrading guide I suppose.

Reply:
I updated the user guide, but I'm unsure where to put this here paragraph. It seems like deprecation warnings for runtime validation problems is a feature we should call out in the release notes, but it's also important when you are upgrading to Gradle 6.0. Adding the same content twice sounds wrong, though. Ideas?

Reply:
I think it needs to go to the upgrade notes, and we should be calling this out as a feature. We can do this by either adding the same content twice, or slightly different content. The one in the upgrading guide should be more focussed on how to get rid of the deprecation warnings while the one in the release notes is more about that you get now helpful information about possibly ill-defined tasks.

Reply:
Is this good now, @wolfs?